core-backend/sandbox/src/funcs.rs

```diff
@@ -1103,7 +1000,7 @@ where
     pub fn out_of_allowance(ctx: &mut Runtime<E>, _args: &[Value]) -> SyscallOutput {
         sys_trace!(target: "syscall::gear", "out_of_allowance");
 
-        ctx.err = ActorSyscallFuncError::Core(ctx.ext.out_of_allowance()).into();
+        ctx.termination_reason = ctx.ext.out_of_allowance().into_termination_reason();
```

Comment:
sync with @ark0f who will remove `out_of_allowance` and `out_of_gas` fns from ext as unused stuff, cause they are always free and infallible

Reply:
Done TODO